pkg/monitor/monitor/util.go

```diff
@@ -586,6 +716,19 @@ func getMonitorVolumes(config *core.ConfigMap, monitor *v1alpha1.TidbMonitor, tc
 		}
 		volumes = append(volumes, tlsPDClient)
 	}
+	if dc != nil && dc.IsTLSClusterEnabled() {
+		defaultMode := int32(420)
+		tlsDMClient := core.Volume{
+			Name: util.DMClusterClientVolName,
+			VolumeSource: core.VolumeSource{
+				Secret: &core.SecretVolumeSource{
+					SecretName:  util.ClusterClientTLSSecretName(dc.Name),
```

Comment:
This should not be the correct client secret name for DM, @lichunzhu

Reply:
fixed, dm cluster works fine when tls enabled